18F/identity-idp
# Redirect user with a pending profile to /account

**Why**: Previously, users would be continuously redirected to the
/verify page if they choose to have their account verified by mail,
which could be confusing.

Verification instructions:
1). Create or sign in with a user via SP (LOA3)
2). Go through all IdV steps
3). Enter your password to confirm profile info
4). Confirm your personal key
5). Land on the `/account` page, instead of on `/verify`
